config/crd/bases/config.scheduling.sigs.k8s.io_coschedulingargs.yaml

```diff
@@ -0,0 +1,90 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.7.0
```

Comment:
There was a KEP url here, can you add it back? (applies elsewhere)

Reply:
This is a generated file.. I'll try to find a way to add it. Meanwhile, it's still in the manifests folder